recipes/flake8-sfs/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "flake8-sfs" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "aeada40b526bf49d443f3f21483705dd8fa958c23809671e2ea8ab8cc1eeb4ab" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python
```

Comment:
Use `python >=3.6` instead.

Reply:
Good catch, that was inconsistent and important.

Reply:
I think you changed `flake8` version instead of `python` by mistake :)

Reply:
Thank you, yes.

While I'm at I, might as well make the requirement for at least flake8 v3 explicit too.